app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,34 @@
+
+  <div class="task-list">
+    <h2>Your Tasks</h1>
+
+      <%if @tasks.count == 0%>
+
+        <div class="wrap">
```

Comment:
Nice wine glass, made me laugh! 

I would suggest using a `section` instead of a div for it however.